docs/source/Controller/C013.rst

```diff
@@ -247,7 +247,7 @@ These messages are just a serialized byte stream of ``struct C013_SensorInfoStru
   {
     byte header = 255;
     byte ID = 3;
-    byte sourceUnit;
+    byte sourcelUnit;

```

Comment:
Why change it back?
I renamed it yesterday as I was under the impression it was a typo.